python/examples/core/1-first-steps.py

```diff
@@ -0,0 +1,264 @@
+"""
+.. _core-tutorial-first-steps:
+
+First steps with metatensor
+===========================
+
+This tutorial explores how data is stored inside metatensor's ``TensorMap``, and how to
+access the associated metadata.
+
+To this end, we will need some data in metatensor format, which for the sake of
+simplicity will be loaded from a file. The code used to generate this file can be found
+below:
+
+.. details:: Show the code used to generate the :file:`spherical-expansion.npz` file
```

Comment:
Maybe add a download that people can download the file? Or do we want them to create this by themselves?

Reply:
Good point! It might be in the sphinx-gallery zip, but I'll add another link